### PR TITLE
Remove hardcoded "/"

### PR DESCRIPTION
**Pull request by rht**
Just small fixes.

Another option, it would be faster to just use strings.Join(elms, "/") and put it in path/path.go, since golang's path does path.Clean of every elements.

### CODE REVIEW

**Comment by rht** [comment]
There is also strings.Split -> path.Segments.

**Comment by rht** [comment]
(in #1984, stating '/' directly is a crime)

**Comment by rht** [comment]
Somehow the tests passed.

File: commands/command.go [hunk 1]
```diff
@@ -143,16 +143,16 @@ func (c *Command) Call(req Request) Response {
...
 	cmds[0] = c
 
 	cmd := c
-	for i, name := range path {
+	for i, name := range pth {
 		cmd = cmd.Subcommand(name)
 
 		if cmd == nil {
-			pathS := strings.Join(path[0:i], "/")
+			pathS := path.Join(pth[0:i])
```
Comment (whyrusleeping) [inline commands/command.go]: I know you didnt add it, but that `[0:i]` not being just `[:i]` is bothering me

File: mfs/ops.go [hunk 2]
```diff
@@ -156,9 +157,9 @@ func Lookup(r *Root, path string) (FSNode, error) {
 
 // DirLookup will look up a file or directory at the given path
 // under the directory 'd'
-func DirLookup(d *Directory, path string) (FSNode, error) {
-	path = strings.Trim(path, "/")
-	parts := strings.Split(path, "/")
+func DirLookup(d *Directory, pth string) (FSNode, error) {
+	pth = strings.Trim(pth, "/")
```
Comment (whyrusleeping) [inline mfs/ops.go]: this could be `path.Clean` no?

**Comment by whyrusleeping** [comment]
LGTM

File: commands/command.go (hunk 1 above)
Comment (rht) [inline commands/command.go]: There are 6 occurences of '\[0:'. Purging them once for all.

File: mfs/ops.go (hunk 2 above)
Comment (rht) [inline mfs/ops.go]: Actually, if the path comes from cli, it should have already been cleaned in commands/cli/parse.go

Comment (whyrusleeping) [inline mfs/ops.go]: ah, thats true. carry on

**Comment by jbenet** [comment]
this LGTM. thanks @rht

**Pull request merged**